### ISSUE TITLE
bug: Missing FifoTopic attributes on FIFO topic

### ISSUE THREAD

**Issue by aleskiewicz** [localstack/localstack#4712]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Creating a FIFO topic with Java SDK 
```
        val topicAttributes = mapOf("FifoTopic" to "true", "ContentBasedDeduplication" to "false")

        val topic: CreateTopicResult? = snsClient.createTopic(
            CreateTopicRequest()
                .withName(name)
                .withAttributes(topicAttributes)
        )
```

After fetching topic attributes with `snsClient.getTopicAttributes("proper_arn")` localstack returns 9 attributes, but "FifoTopic" and "ContentBasedDeduplication" are missing

### Expected Behavior

FIFO attributes should be returned.

https://docs.aws.amazon.com/sns/latest/api/API_GetTopicAttributes.html

### How are you starting LocalStack?

Custom (please describe below)

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

```
@Testcontainers
public open class LocalstackTest {
    public companion object {
        private var localstackImage: DockerImageName = DockerImageName.parse("localstack/localstack:0.12.18")

        @Container
        public var localstack: LocalStackContainer = LocalStackContainer(LocalstackTest.localstackImage)
            .withServices(
                LocalStackContainer.Service.S3,
                LocalStackContainer.Service.SNS,
                LocalStackContainer.Service.SQS
            )
    }
}
```

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

```
        val snsClient = AmazonSNSClientBuilder
            .standard()
            .withEndpointConfiguration(localstack.getEndpointConfiguration(LocalStackContainer.Service.S3))
            .withCredentials(localstack.defaultCredentialsProvider)
            .build()

        val topicAttributes = mapOf("FifoTopic" to "true", "ContentBasedDeduplication" to "false")

        logger.info("Created topic: {}", topic)
        val topicAttributesResult = snsClient.getTopicAttributes(topic.topicArn)
        logger.info("Topic attributes: {}", topicAttributesResult.attributes)
```

### Environment

```markdown
- OS: Ubuntu 21.04
- LocalStack: latest
```

**Comment by whummer** [comment]
Thanks @aleskiewicz . This should be fixed in spulec/moto#4444 - can you please pull the `latest` Docker image and give it another try? Please let us know if the problem persists.. Thanks!

**Comment by aleskiewicz** [comment]
Thanks @whummer ! I confirm, using `latest` tag everything works fine.
